cpp/modmesh/transform/fourier.hpp

```diff
@@ -100,6 +100,8 @@ class FourierTransform
     template <template <typename> typename T1, typename T2>
     static void dft(SimpleArray<T1<T2>> const & in, SimpleArray<T1<T2>> & out)
     {
+        out.fill(0);
```

Comment:
Why initialize the array using a distinct loop but not in the beginning of the outer loop over `i` below?  `out[i]` is read at line 112 so the initialization right before the loop over `j` is logically clearer.

Reply:
I see. Thanks.